private/pkg/git/repository.go

```diff
@@ -212,19 +194,18 @@ func (r *repository) ForEachCommit(f func(Commit) error, options ...ForEachCommi
 func (r *repository) ForEachTag(f func(string, Hash) error) error {
 	seen := map[string]struct{}{}
 	// Read unpacked tag refs.
-	dir := path.Join(r.gitDirPath, "refs", "tags")
+	dir := normalpath.Join(r.gitDirPath, "refs", "tags")
 	if err := filepathextended.Walk(dir, func(path string, info fs.FileInfo, err error) error {
 		if err != nil {
 			return err
 		}
 		if !info.Mode().IsRegular() {
 			return nil
 		}
-		tagName, err := filepath.Rel(dir, path)
+		tagName, err := normalpath.Rel(dir, path)
 		if err != nil {
 			return err
 		}
-		tagName = normalpath.Normalize(tagName)
```

Comment:
No need to normalize the received `tagName` if we used `normalpath.Rel` instead of `filepath.Rel`.